lock-api/src/main/java/com/palantir/lock/client/ProfilingTimelockService.java

```diff
@@ -169,6 +171,21 @@ public long currentTimeMillis() {
         return runTaskTimed("currentTimeMillis", delegate::currentTimeMillis);
     }
 
+    @ReviewedRestrictedApiUsage
+    @Override
+    public AcquireNamedMinTimestampLeaseResult acquireNamedMinTimestampLease(
+            String timestampName, int numFreshTimestamps) {
+        return runTaskTimed(
+                "acquireNamedMinTimestampLease",
+                () -> delegate.acquireNamedMinTimestampLease(timestampName, numFreshTimestamps));
+    }
+
+    @ReviewedRestrictedApiUsage
+    @Override
+    public long getMinLeasedNamedTimestamp(String timestampName) {
+        return runTaskTimed("getMinLeasedNamedTimestamp", () -> delegate.getMinLeasedNamedTimestamp(timestampName));
+    }
+
```

Comment:
Usage is controlled by us, instrumenting these is useful then (unlike the arbitrary user ones, or the blocking lock ones)